museek/util/time_analysis.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from datetime import datetime, timedelta
+import ephem
+from museek.time_ordered_data import TimeOrderedData
+
+class TimeAnalysis:
```

Comment:
We'll need unittests in `test/museek/util/TestTimeAnalysis`
Have you done that before? If not, let me know...

Reply:
I have not done this before.